dev-docs/bidders/taboola.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bidder
+title: Taboola
+description: Prebid Taboola Bidder Adapter
+pbjs: true
+biddercode: taboola
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner
+gvl_id: 42
+prebid_member: true
```

Comment:
I do not see Taboola listed on this page, https://prebid.org/membership/member-directory/